File: src/Api/Model/Languageforge/Lexicon/Config/LexConfigMultiText.php
<?php

namespace Api\Model\Languageforge\Lexicon\Config;

use Api\Model\Shared\Mapper\ArrayOf;

class LexConfigMultiText extends LexConfig
{
    public function __construct()
    {
        parent::__construct();
        $this->type = LexConfig::MULTITEXT;
        $this->displayMultiline = false;
        $this->width = 20;
        $this->inputSystems = new ArrayOf();
    }

    /** @var int */
    public $width;

    /** @var ArrayOf */
    public $inputSystems;

    /** @var boolean */
    public $displayMultiline;
}

File: src/netcore-api/SIL.XForge.WebApi.Server/Models/Lexicon/LexConfigMultiText.cs
using System.Collections.Generic;

namespace SIL.XForge.WebApi.Server.Models.Lexicon
{
    public class LexConfigMultiText : LexConfig
    {
        public LexConfigMultiText(string label, string inputSystem, bool hideIfEmpty = false)
        {
            Type = MultiText;
            Label = label;
            HideIfEmpty = hideIfEmpty;
            InputSystems.Add(inputSystem);
        }

        public LexConfigMultiText()
        {
            Type = MultiText;
        }

        public int Width { get; set; } = 20;
        public List<string> InputSystems { get; protected set; } = new List<string>();
        public bool DisplayMultiline { get; set; }
    }
}
